portfolio/src/main/webapp/script.js

```diff
@@ -42,10 +42,15 @@ function addRandomGreeting() {
  * Displays the right sidebar
  */
 let isSidebarOpen = false;
+let isBottombarOpen = false;
 function toggleSidebar() {
     let button = document.getElementById("commentbutton");
     let el = document.getElementById("sidebar");
     if (isSidebarOpen === false){
+        if (isBottombarOpen){
+            toggleBottomBar();
+            isBottombarOpen = !isBottombarOpen;
```

Comment:
you know isBottombarOpen is true at this point (from the if statement), so just write:
```
isBottombarOpen = false;
```